netcdftime/netcdftime.py

```diff
@@ -119,6 +120,12 @@ def JulianDayFromDate(date, calendar='standard'):
     # adjust for Julian calendar if necessary
     jd = jd + B
 
+    if domc:
+        # MC - Add a small offset (proportional to Julian date) for correct re-conversion.
+        eps = 2.22e-016 # ~ Fortran epsilon(1.0) in double precision
```

Comment:
I would definitely use `np.finfo(float).eps` instead. It's really not an expensive function to calculate (it's cached after all), and using it has the advantage of making the purpose 100% obvious without a comment.